libweb3jsonrpc/UnixSocketServer.h

```diff
@@ -45,7 +46,7 @@ class UnixDomainSocketServer: public IpcServerBase<int>
 	size_t Read(int _connection, void* _data, size_t _size) override;
 
 	sockaddr_un m_address;
-	int m_socket = 0;
+	std::atomic<int> m_socket{0};
```

Comment:
I think atomics are always initialized to default value (read 0). So explicit 0 is not needed.

Reply:
They are not http://en.cppreference.com/w/cpp/atomic/atomic/atomic